packages/api/src/qm/ui.ts

```diff
@@ -263,6 +263,11 @@ export interface InputResult<T> {
    * answer value
    */
   result?: T;
+
+  /**
+   * resolved option list
+   */
+  options?: StaticOptions;
```

Comment:
What's the purpose of this property? Can we use `result` directly?

Reply:
The options are resolved for dynamicOptions. Result is the output that user selected. The reason to return the resolved options is to pass the selected `OptionItem` object into `onDidSelection` callback.